python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -132,6 +132,20 @@ $defs:
               type: bool
               default: false
               description: Enable PTP.
+            role:
+              type: str
+              description: EOS default is `dynamic`.
```

Comment:
Since this is applied on a link with two ends, it would not make sense to use "master" on both ends. We need to change this to a list similar to the ip_addresses etc. and then apply it on each device based on the index.

Reply:
Fixed.